docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -20,8 +20,20 @@ The version numbers can be found in these Perl modules:
  * zonemaster-engine - [Engine.pm](https://github.com/zonemaster/zonemaster-engine/blob/master/lib/Zonemaster/Engine.pm)
  * zonemaster-cli - [CLI.pm](https://github.com/zonemaster/zonemaster-cli/blob/master/lib/Zonemaster/CLI.pm)
  * zonemaster-backend - [Backend.pm](https://github.com/zonemaster/zonemaster-backend/blob/master/lib/Zonemaster/Backend.pm)
+
+The GUI has no Perl. Update the following files:
+
  * zonemaster-gui - [GUI.pm](https://github.com/zonemaster/zonemaster-gui/blob/master/lib/Zonemaster/GUI.pm)
 
+ * zonemaster-gui - [Installation.md](https://github.com/zonemaster/zonemaster-gui/blob/master/docs/Installation.md):
+   The version is part of the download path (a diretory). It can be repeated several times.
+ * zonemaster-gui - [package.json](https://github.com/zonemaster/zonemaster-gui/blob/master/package.json):
+   In the top of the file, the version is given after "version".
+ * zonemaster-gui - [environment.prod.ts](https://github.com/zonemaster/zonemaster-gui/blob/master/src/environments/environment.prod.ts):
+   Under "client-info", "version" should point at the version number.
+ * zonemaster-gui - [environment.ts](https://github.com/zonemaster/zonemaster-gui/blob/master/src/environments/environment.ts):
```

Comment:
Only used for development purpose, but yes, for a new version, we should update this "version" number too.